webapp/i18n/en.json

```diff
@@ -818,6 +818,9 @@
   "admin.sql.keyTitle": "At Rest Encrypt Key:",
   "admin.sql.maxConnectionsDescription": "Maximum number of idle connections held open to the database.",
   "admin.sql.maxConnectionsExample": "E.g.: \"10\"",
+  "admin.sql.connectionTimeoutTitle": "Connection Timeout:",
+  "admin.sql.connectionTimeoutDescription": "The number of seconds a query can be executing for before being cancelled.",
```

Comment:
Is this a connection timeout where the system gives up trying to connect after a specified amount of time, or is it for after the connection is made and the system gives up waiting for a response after a specified amount of time?